discovery/linode/linode.go

```diff
@@ -143,6 +156,50 @@ func NewDiscovery(conf *SDConfig, logger log.Logger) (*Discovery, error) {
 }
 
 func (d *Discovery) refresh(ctx context.Context) ([]*targetgroup.Group, error) {
+	needsRefresh := true
+
+	if d.lastResults != nil && d.eventPollingEnabled {
+		// Update the last refresh timestamp - better to have some overlap than miss something.
+		d.lastRefreshTimestamp = time.Now().UTC()
+
+		// Check to see if there have been any events. If so, refresh our data.
+		opts := linodego.NewListOptions(1, fmt.Sprintf(filterTemplate, d.lastRefreshTimestamp.Format("2006-01-02T15:04:05")))
+		events, err := d.client.ListEvents(ctx, opts)
```

Comment:
This seems like it will only fetch the events in the last second?

Reply:
Addressed in f61624850c064dc0b7933fcb70c50adeaf383462